Docs/Practitioners/CodedDescription.md

```diff
@@ -0,0 +1,8 @@
+# CodedDescription
+
+Holds a coded description
```

Comment:
Describing something in terms of itself is not that helpful. This type is simply a wrapper around a code and a user-friendly description of the code.

Reply:
The more I think about it the more I believe we should move the types to their own folder called something like Types. Types can be reused across resources potentially.